src/System.AppContext/pkg/any/System.AppContext.pkgproj

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <!-- there is no "jit" rid, so we use any and "aot" will override -->
```

Comment:
Is this comment correct? ANY could apply for both JIT and AOT assets. I agree if we have an AOT specific library then we will override with AOT RID but not everything that is ANY is JIT only.